exported_data_bucket.tf

```
@@ -1,24 +1,56 @@
+# This bucket is used to store a JSON file containing all exported
+# assessment data to be imported into the CyHy database.
 resource "aws_s3_bucket" "exported_data" {
-  # This bucket is used to store a JSON file containing all exported
-  # assessment data to be imported into our AWS database.
-  # Note that in production terraform workspaces, the string '-production' is
+  # Note that in production Terraform workspaces, the string '-production' is
   # appended to the bucket name.  In non-production workspaces,
   # '-<workspace_name>' is appended to the bucket name.
-  bucket = local.production_workspace ? format("%s-production", var.assessment_data_s3_bucket) : format("%s-%s", var.assessment_data_s3_bucket, terraform.workspace)
-
-  server_side_encryption_configuration {
-    rule {
-      apply_server_side_encryption_by_default {
-        sse_algorithm = "AES256"
-      }
-    }
-  }
+  bucket = format("%s-%s", var.assessment_data_s3_bucket, local.production_workspace ? "production" : terraform.workspace)
 
   tags = {
     "Name" = "Exported Assessment Data"
   }
 
   lifecycle {
+    ignore_changes = [
+      # This should be removed when we upgrade the Terraform AWS provider to
+      # v4. It is necessary to use with the back-ported resources in v3.75 to
```

Consistency - we don't always hyphenate this word, even in the files in this PR:
```suggestion
      # v4. It is necessary to use with the backported resources in v3.75 to
```